src/android/aidl/example-service/server.md

```diff
@@ -0,0 +1,36 @@
+# AIDL Server
+
+Finally, we can create a server which exposes the service:
+
+_birthday_service/src/server.rs_:
+
+```rust,ignore
+{{#include ../birthday_service/src/server.rs:main}}
+```
+
+_birthday_service/Android.bp_:
+
+```javascript
+{{#include ../birthday_service/Android.bp:birthday_server}}
+```
+
+<details>
+
+The process for taking a user-defined service implementation (in this case the
+`BirthdayService` type, which implements the `IBirthdayService`) and starting it
+as a binder service has multiple steps, and may appear more complicated than
+students are used to if they've used Binder from C++ or another language.
+Explain to students why each step is necessary.
+
+1. Create an instance of your service type (`BirthdayService`).
+1. Wrap the service object in corresponding `Bn*` type (`BnBirthdayService` in
+   this case). This type is generated by Binder and provides the common binder
```

Comment:
Minor spelling nit: "Binder" and "binder" in the same sentence? Was that intentional?

Reply:
Nah, that was an accident. I've normalized it to "Binder" everywhere.